src/CodeGen/ODataT4CodeGenerator.ttinclude

```diff
@@ -48,6 +48,7 @@ THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
             UseDataServiceCollection = this.UseDataServiceCollection,
             TargetLanguage = this.TargetLanguage,
             EnableNamingAlias = this.EnableNamingAlias,
+			TempFilePath = this.TempFilePath,

```

Comment:
nit for consistency: should be white space

Reply:
Fixed.